internal/cmd/asyncapi/command_import.go

```diff
@@ -0,0 +1,444 @@
+			return true
+		}
+	}
+	return false
+}
+
+func (c *command) addTopic(details *accountDetails, topicName string, description string, kb KafkaBinding, overwrite bool) (bool, error) {
+	topicConfigs := []kafkarestv3.CreateTopicRequestDataConfigs{}
+	updateConfigs := []kafkarestv3.AlterConfigBatchRequestDataData{}
+	for configName, configValue := range kb.XConfigs {
+		value := configValue
+		topicConfigs = append(topicConfigs, kafkarestv3.CreateTopicRequestDataConfigs{
+			Name:  configName,
+			Value: *kafkarestv3.NewNullableString(&value),
+		})
+		if isModifiableConfig(configName) {
+			updateConfigs = append(updateConfigs, kafkarestv3.AlterConfigBatchRequestDataData{
+				Name:  configName,
+				Value: *kafkarestv3.NewNullableString(&value),
+			})
+		}
+	}
+	topicRequestData := kafkarestv3.CreateTopicRequestData{
+		TopicName: topicName,
+		Configs:   &topicConfigs,
+	}
+	// Check if topic already exists
+	for _, topics := range details.topics {
+		if topics.TopicName == topicName {
+			// Topic already exists
+			log.CliLogger.Info("Topic is already present.")
+			if !overwrite {
+				// Do not overwrite existing topic. Move to the next topic.
+				return true, nil
+			}
+			// Overwrite topic configs
+			log.CliLogger.Info("Overwriting topic configs")
+			_, err := c.kafkaRest.CloudClient.UpdateKafkaTopicConfigBatch(details.clusterId, topicName, kafkarestv3.AlterConfigBatchRequestData{Data: updateConfigs})
+			if err != nil {
+				return true, fmt.Errorf("unable to update topic config delete.retention.ms: %v", err)
+			}
+			if description != "" {
+				err = addTopicDescription(details.srClient, details.srContext, details.clusterId+":"+topicName, description)
+			}
+			return true, err
+		}
+	}
+	log.CliLogger.Infof("Topic not found. Adding a new topic: %s", topicName)
+	_, httpResp, err := c.kafkaRest.CloudClient.CreateKafkaTopic(details.clusterId, topicRequestData)
+	if err != nil {
+		restErr, parseErr := kafkarest.ParseOpenAPIErrorCloud(err)
+		if parseErr == nil && restErr.Code == ccloudv2.BadRequestErrorCode {
+			// Print partition limit error w/ suggestion
+			if strings.Contains(restErr.Message, "partitions will exceed") {
+				return false, errors.NewErrorWithSuggestions(restErr.Message, errors.ExceedPartitionLimitSuggestions)
+			}
+		}
+		return false, kafkarest.NewError(c.kafkaRest.CloudClient.GetUrl(), err, httpResp)
+	}
+	log.CliLogger.Infof("Added topic %s to cluster %s.\n", topicName, details.clusterId)
+
+	if description != "" {
+		err = addTopicDescription(details.srClient, details.srContext, details.clusterId+":"+topicName, description)
+	}
+	return false, err
+}
+
+func addTopicDescription(srClient *srsdk.APIClient, srContext context.Context, qualifiedName string, description string) error {
+	atlasEntity := srsdk.AtlasEntityWithExtInfo{
+		ReferredEntities: nil,
+		Entity: srsdk.AtlasEntity{
+			Attributes: map[string]interface{}{"description": description, "qualifiedName": qualifiedName},
```

Comment:
Please multi-line structs and maps that contain more than one item.
```suggestion
			Attributes: map[string]any{"description": description, "qualifiedName": qualifiedName},
```